services/core-web/src/components/mine/Projects/ProjectSummary.tsx

```diff
@@ -130,20 +137,42 @@ export const ProjectSummary: FC = () => {
         payload,
         message
       )
-    )
-      .then(async () => {
-        await dispatch(
-          updateProject(
-            { projectGuid },
-            { mrc_review_required: payload.mrc_review_required, contacts: payload.contacts },
-            "Successfully updated project.",
-            false
+    );
+
+    await dispatch(
+      updateProject(
+        { projectGuid },
+        {
+          mrc_review_required: payload.mrc_review_required,
+          contacts: payload.contacts,
+        },
+        "Successfully updated project.",
+        false
+      )
+    );
+
+    await handleFetchData();
+    if (
+      tab === PROJECT_SUMMARY_WITH_AMS_SUBMISSION_SECTION &&
+      amsFeatureEnabled &&
+      projectSummaryResponse
+    ) {
+      const { data } = projectSummaryResponse;
+      const authorizations = data?.authorizations;
```

Comment:
should probably be `data?.authorizations ?? []` for the array operations below

Reply:
Thanks, this has been adjusted now